### PR TITLE
RESTWS-813:Strings should not be compared by using == or !=

### PR DESCRIPTION
**Pull request by hillarykimo**
## Description of what I changed
I changed the operators from   `== or != `  to ` equals(Object) method ` on this files
https://github.com/openmrs/openmrs-module-webservices.rest/blob/2.28.0/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleResource1_8.java#L210
https://github.com/openmrs/openmrs-module-webservices.rest/blob/2.28.0/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/TaskDefinitionResource1_8.java#L157
https://github.com/openmrs/openmrs-module-webservices.rest/blob/2.28.0/omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java#L1041

## Issue I worked on
https://issues.openmrs.org/browse/RESTWS-813

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by HerbertYiga** [comment]
> https://github.com/openmrs/openmrs-module-webservices.rest/blob/2.28.0/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleResource1_8.java#L210
> https://github.com/openmrs/openmrs-module-webservices.rest/blob/2.28.0/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/TaskDefinitionResource1_8.java#L157
> https://github.com/openmrs/openmrs-module-webservices.rest/blob/2.28.0/omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java#L1041

Thanks @hillarykimo  no need of adding this

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java [hunk 1]
```diff
@@ -228,7 +228,7 @@ private void startModules(Collection<Module> modules, ServletContext servletCont
 		}
 		
 		for (Module module : modules) {
-			if (moduleFactoryWrapper.isModuleStopped(module)) {
+			if ((moduleFactoryWrapper.isModuleStopped(module)&& module.getModuleId()).equal( existingModule.getModuleId())) {
```
Comment (HerbertYiga) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java:231]: aren't we changing the meaning of the original line by adding this change?

Comment (hillarykimo) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java:231]: @HerbertYiga  i have made changes  kindly check in commit  thank you

**Comment by dkayiwa** [comment]
@hillarykimo did you successfully compile these changes with `mvn clean install`?

**Comment by jwnasambu** [comment]
@dkayiwa his laptop broke down and he is trying to fix it.

**Pull request closed without merging**